src/components/AllReviews.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import * as api from "./api";
+import { ReviewBox } from "./ReviewBox";
+
+export const AllReviews = () => {
+    const [reviews, setReviews] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
You've got your loading state that would done use.
COuld we get some feedback for when its loading on the page?